executor/executor.go

```diff
@@ -1859,6 +1923,7 @@ func (e *ExistsExec) Next() (*Row, error) {
 }
 
 // MaxOneRowExec checks if a query returns no more than one row.
```

Comment:
MaxOneRowExec checks if the number of rows that a query returns are more than one.